packages/features/calendars/DatePicker.tsx

```diff
@@ -138,6 +139,21 @@ const Days = ({
     ? availableDates(props.includedDates)
     : props.includedDates;
 
+  useEffect(() => {
+    if (selected && selected.date()) {
+      if (selected.format("YYYY-MM") === browsingDate.format("YYYY-MM")) return;
```

Comment:
This is really difficult for someone new to understand what's going on.
Would it make sense to abstract it out in `areWeOnTheSameMonth` function ?